src/Faker/Provider/fr_FR/Company.php

```diff
@@ -13,7 +13,6 @@ class Company extends \Faker\Provider\Company
         '{{lastName}} {{companySuffix}}',
         '{{lastName}} {{lastName}} {{companySuffix}}',
         '{{lastName}}',
-        '{{lastName}}',
```

Comment:
Duplicates have a function in format lists: making the duplicated patter occur more often. Please remove this change.